### ISSUE TITLE
Host info doesn't show Docker data/metadata space sizes

### ISSUE THREAD

**Issue by gordontyler** [rancher/rancher#2693]
The Host page doesn't show any information about the Docker engine's data and metadata spaces. Relevant output from `docker info`:

```
Storage Driver: devicemapper
 Pool Name: docker-253:0-202318689-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: xfs
 Data file: /dev/vg-docker/data
 Metadata file: /dev/vg-docker/metadata
 Data Space Used: 16.66 GB
 Data Space Total: 160 GB
 Data Space Available: 143.3 GB
 Metadata Space Used: 14.01 MB
 Metadata Space Total: 10.74 GB
 Metadata Space Available: 10.72 GB
```

**Comment by gordontyler** [comment]
From a host configured with loop-lvm devicemapper:

```
Storage Driver: devicemapper
 Pool Name: docker-253:0-202318689-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: xfs
 Data file: /dev/vg-docker/data
 Metadata file: /dev/vg-docker/metadata
 Data Space Used: 16.66 GB
 Data Space Total: 160 GB
 Data Space Available: 143.3 GB
 Metadata Space Used: 14.01 MB
 Metadata Space Total: 10.74 GB
 Metadata Space Available: 10.72 GB
```

**Comment by gordontyler** [comment]
From a host configured with loopback devicemapper (default on Centos 7):

```
Storage Driver: devicemapper
 Pool Name: docker-253:0-201328679-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 107.4 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 2.616 GB
 Data Space Total: 107.4 GB
 Data Space Available: 36.17 GB
 Metadata Space Used: 2.666 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.145 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
```

**Comment by will-chan** [comment]
Closing due to inactivity.
